resources/web/wwi/animation.js

```diff
@@ -0,0 +1,204 @@
+/* global ResourceManager */
+'use strict';
+
+function Animation(url, scene, view, gui, loop) {
+  this.url = url;
+  this.scene = scene;
+  this.view = view;
+  this.gui = gui === undefined || gui === 'play' ? 'play' : 'pause';
+  this.loop = loop === undefined ? true : loop;
+  this.sliding = false;
+  this.onReady = null;
+};
+
+Animation.prototype = {
+  constructor: Animation,
+
+  init: function(onReady) {
+    var that = this;
+    this.onReady = onReady;
+    var xmlhttp = new XMLHttpRequest();
+    xmlhttp.open('GET', this.url, true);
+    xmlhttp.overrideMimeType('application/json');
+    xmlhttp.onreadystatechange = function() {
```

Comment:
Yes I know.
But I'm not using ECS6 because I had the impression that it is not widely supported by all the browser.

Reply:
I think ECS6 can be used without fear. It is well supported on every up-to-date browser: https://kangax.github.io/compat-table/es6/
I think we should rather think about using Babel in case of compatibility issues.
Using this workflow, the source code will be much cleaner.